### DISCUSSION TITLE
How to include xtraplatform-core library into ldproxy from local repository

### DISCUSSION THREAD

**Discussion by JoeChen7702** [ldproxy/ldproxy#983] [Setup]
Hi there,

Regarding our data source of Web Feature Service, we need to modify the URL parameter when making a requirement to Web Feature Service. Also, We need to modify the srsName parameter to http://www.opengis.net/def/crs/EPSG/0/2326 as well. 
To achieve this, I have already packaged the xtraplatform-core library but did not know how to include it in ldproxy. 
Could you please guide me on how to modify the configuration at xtraplatform.gradle in ldproxy?

Joe

**Comment by azahnen** [comment]
See #999.
